cdap-ui/app/directives/dag/my-dag.html

```diff
@@ -74,12 +74,12 @@
                 ng-mouseenter="MyDAGController.nodeMouseEnter(node)"
                 ng-mouseleave="MyDAGController.nodeMouseLeave(node)">
             <div class="error-node-notification"
-                 ng-if="node.error"
-                 tooltip="{{node.error.message}}"
+                 ng-if="node.errorCount > 0"
+                 tooltip="Please see node configuration panel"
```

Comment:
Do we still need a tooltip then? If we are not giving specific message, I don't see a point in having a tooltip..